### PR TITLE
Issue #50

### PR DESCRIPTION
**Pull request by petekeller2**
isPlural and isSingular functions added

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -618,6 +618,32 @@ var PLURAL_TESTS = [
...
+ *
+ * @type {Array}
+ */
+var IS_SINGULAR_TESTS = [
+  ['whisky', true],
+  ['echos', false],
+  ['bluffs', false],
+  ['incentive', true],
+  ['sleeve', true]
+];
```
Comment (blakeembrey) [inline test.js]: Couldn't these just use the existing test suite since the left is always singular and the right is always the plural?

**Comment by blakeembrey** [comment]
There's probably a more performant way to write this if it's going to be implemented in the library itself. Should be easy to alter `replaceWord` to instead return `true` or `false` - no need to do some steps if we're just doing detection. If you're willing to do that, feel free - otherwise I'll keep this here as I consider it and alter it in the future.

File: test.js (hunk 1 above)
Comment (petekeller2) [inline test.js]: Yes. I'm going to remove them in the next commit

**Comment by petekeller2** [comment]
I'll try to do that

**Comment by Redsandro** [comment]
I am looking for this. Any update on this since the may 1st update @blakeembrey ?

**Comment by blakeembrey** [comment]
@Redsandro You can use the original approach for now until this is released.

**Comment by blakeembrey** [comment]
If there's more external feedback, I can look into this earlier. I think there's a faster approach here since this PR is still doing a lot of word mutation that wouldn't be necessary to check if it's a plural. I also think it's probably worth not checking irregulars here.

**Comment by Redsandro** [comment]
@blakeembrey said:
> @Redsandro You can use the original approach for now until this is released.

A premise in the realm of moot. I'm checking for singulars and plurals in a lot of places, and I came to the conclusion that it would make the most sense if this library would handle that. 

If nothing else, purely for sugar. :+1:

**Comment by Redsandro** [comment]
Nice, thank you. :+1:

**Pull request closed without merging**